docs/concepts/configuration.md

```diff
@@ -92,6 +131,26 @@ Nevertheless, default values can be provided by environment files and can later
 
 Everything managed in the NgRx state is accumulated on the server side and sent to the client side with the initial HTML response.
 
+To access these properties we provide the `StatePropertiesService` that takes care of retrieving the configuration either from the configuration state, an environment variable or the environment.ts (in that order).
+
+```typescript
+import { StatePropertiesService } from 'ish-core/utils/state-transfer/state-properties.service';
+
+  constructor(
+    private stateProperties: StatePropertiesService,
+  ) {}
+
+this.stateProperties.getStateOrEnvOrDefault<string>('PROPERTY_KEY', 'propertyKey')
+```
```

Comment:
please link to method in the code instead of supplying snippet. Maintainability above all!

Reply:
You would prefer a link to https://github.com/intershop/intershop-pwa/blob/develop/src/app/core/utils/state-transfer/state-properties.service.ts instead of this code snipped on how to actually use it?

Reply:
Yes, if you don't think the method is properly (and minimally) documented, please improve it there 😉